### ISSUE TITLE
No apps are getting installed

### ISSUE THREAD

**Issue by GamingNerdLeith** [ChrisTitusTech/winutil#1494]
**Describe the bug**
Whenever you try to install applications an error "No applicable installer found; see logs for more details." appears. This so far has appeared with the following apps:

**"OBSProject.OBSStudio", "KeePassXCTeam.KeePassXC", "JanDeDobbeleer.OhMyPosh", "OpenJS.NodeJS", "Notepad++.Notepad++",
"VSCodium.VSCodium", "LizardByte.Sunshine", "GOG.Galaxy",  "Microsoft.PowerShell", "TorProject.TorBrowser", "SomePythonThings.WingetUIStore", "Nvidia.GeForceNow", "Nilesoft.Shell", "7zip.7zip", "Unity.UnityHub", "Discord.Discord",
"EpicGames.EpicGamesLauncher", "RARLab.WinRAR", "BlenderFoundation.Blender", "Microsoft.Teams"**

**To Reproduce**
1. Go to 'Start'
2. Type "Terminal"
3. Type "iwr -useb https://christitus.com/win | iex" to open WinUtil
4. Press Enter
5. Click on the apps listed above 
6. Click on 'Get Installed'
7. Click on 'Installed Selected'
8.  See Error

**Expected behavior**
The applications will be installed into Windows 11 without error. 

**Screenshots**
![image](https://github.com/ChrisTitusTech/winutil/assets/67237948/ac2fa2a4-0b5b-4b43-a3bd-dd64ca0428d7)

**Additional context**
Processor	Intel(R) Core(TM) i5-10300H CPU @ 2.50GHz   2.50 GHz
Installed RAM	16.0 GB (15.9 GB usable)
System type	64-bit operating system, x64-based processor
Edition	Windows 11 Pro
Version	23H2
Installed on	‎28/‎01/‎2024
OS build	22631.3085
Experience	Windows Feature Experience Pack 1000.22684.1000.0

Tried to reinstall winget with the issue not being resolved.

**Comment by RocGod** [comment]
I have this issue as well, but it is not applied to all the applications. Some apps can still be installed.

**Comment by TheFlano23** [comment]
Same issue here. Tried following fixes/workarounds on other issues but none have worked so far. On Windows 10 22H2

**Comment by tegaidogun** [comment]
Same issue here, it seems to be from the `scope=machine` stuff in the scripts.

This thing is really annoying me.

**Comment by Nathan-Shea** [comment]
> Same issue here, it seems to be from the `scope=machine` stuff in the scripts.
> 
> This thing is really annoying me.

Agreed, I checked the winget logs and ran the command minus the scope (and silent) and it's installing fine. 

`winget install -e --accept-source-agreements --accept-package-agreements LibreWolf.LibreWolf OBSProject.OBSStudio Rustlang.Rust.MSVC ********* 9PF4KZ2VN4W9`

I've attached my log in case there is anything useful in there.

[WinGet-2024-01-31-03-54-31.325.log](https://github.com/ChrisTitusTech/winutil/files/14093555/WinGet-2024-01-31-03-54-31.325.log)

**Comment by possumdoe** [comment]
TLDR: Even the developers of `winget` are trying to get things working properly with this `scope` problem

After a quick dive I found this article on Microsoft Learn: https://learn.microsoft.com/en-us/windows/package-manager/winget/troubleshooting#scope-for-specific-user-vs-machine-wide and a related issue on Github https://github.com/microsoft/winget-cli/issues/3240

It's even in the list of things that could justify a bump to a 2.0 release of `winget`: https://github.com/microsoft/winget-cli/issues/2128
